### PR TITLE
Adding side nav menu to the application

### PR DESCRIPTION
**Pull request by natron2244**
Note: I am note testing the routes in the sidebar

### CODE REVIEW

File: src/app/dg-nav/dg-nav.component.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.sidenav-container {
```
Comment (LilBiggs) [inline src/app/dg-nav/dg-nav.component.css]: only .less files should be saved.

File: src/app/dg-nav/dg-nav.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+import { fakeAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+// TODO: Actaully test the routes
```
Comment (LilBiggs) [inline src/app/dg-nav/dg-nav.component.spec.ts]: Remove //TODO or create ticket to implement

Comment (natron2244) [reply src/app/dg-nav/dg-nav.component.spec.ts]: Created https://github.com/natron2244/DogTracker/issues/4 and update comments

File: src/app/dg-nav/dg-nav.component.css (hunk 1 above)
Comment (natron2244) [reply src/app/dg-nav/dg-nav.component.css]: Updated to less file and bem style

**Comment by natron2244** [comment]
@LilBiggs Address all review comments. Good to merge pull request to master?

**Review by LilBiggs — APPROVED** [review]

**Pull request merged**